src/backoffice/core/property-editors/uis/toggle/property-editor-ui-toggle.element.ts

```diff
@@ -26,18 +26,11 @@ export class UmbPropertyEditorUIToggleElement extends UmbLitElement implements U
 	_showLabels?: boolean;
 
 	@property({ type: Array, attribute: false })
```

Comment:
I would say we can remove the Type here. As its being used for attribute value conversion. As attribute is disabled its not needed anymore :-)

Reply:
This applied to multiple places, i just didn't want to write a comment on all of them. :-) I hope that's cool

Reply:
There's a stack of places in fact - 80+ where we declare type but attribute is false. I can clean those up in a separate PR rather than mixing in to this one.

Reply:
cool :-)